packages/deprecation-crawler/src/lib/models.ts

```diff
@@ -62,7 +62,7 @@ export interface RawDeprecation {
 }
 
 export interface CrawledRelease {
-  version: string;
+  gitTag: string;
```

Comment:
Could it be that this property isn't used anymore?
(we moved it to the `Deprecation` interface).